### DISCUSSION TITLE
How to distinguish between VEX and SBOM

### DISCUSSION THREAD

**Discussion by Razikus** [CycloneDX/specification#286] [Q&A] [answered]
Hello

I have following problem - I want to apply VEX to existing SBOM

How can I know that user applies VEX, but not SBOM?
What differentiate it?

Thanks

**Comment by stevespringett** [comment, marked as answer]
A VEX typically will not have any inventory - no components or services. It should only consist of vulnerabilities with the analysis node fully populated. A VEX would also typically use compositions and would indicate the aggregate as being incomplete, vs a VDR which would have an aggregate as complete.
